tests/unit/cli/test_llm_report_unit.py

```
@@ -0,0 +1,345 @@
+                "file_path": "stub.md",
+            },
+        }
+
+
+class _StubGeneratorValidationOnly:
+    """Stub generator for validating --validate-only flow."""
+
+    def __init__(self) -> None:
+        self.validate_calls = 0
+
+    def validate_prerequisites(self, provider: str) -> dict[str, object]:
```

Ruff (0.14.1)

Unused method argument: `provider`

(ARG002)